File: widget/src/main/java/ru/kest/trainswidget/data/FieldDataStorage.java
package ru.kest.trainswidget.data;

import android.location.Location;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import ru.kest.trainswidget.model.domain.TrainThread;

import java.util.List;

/**
 * Created by KKharitonov on 07.01.2016.
 */
@Getter(AccessLevel.PACKAGE) @Setter(AccessLevel.PACKAGE) @AllArgsConstructor(suppressConstructorProperties = true)
public class FieldDataStorage {

    private List<TrainThread> trainsFromHomeToWork;
    private List<TrainThread> trainsFromWorkToHome;
    private Location lastLocation;
    private TrainThread notificationTrain;


}

File: widget/src/main/java/ru/kest/trainswidget/data/FieldDataStorage.kt
package ru.kest.trainswidget.data

import android.location.Location
import ru.kest.trainswidget.model.domain.TrainThread

/**
 * FieldDataStorage Entity
 *
 * Created by KKharitonov on 07.01.2016.
 */
data class FieldDataStorage(
        var trainsFromHomeToWork: List<TrainThread>,
        var trainsFromWorkToHome: List<TrainThread>,
        var lastLocation: Location?,
        var notificationTrain: TrainThread?
) {

}
